ABOUT.md

```diff
@@ -1,9 +1,106 @@
+ [Tapkee](http://tapkee.lisitsyn.me/), 
+ [SLEP](http://www.public.asu.edu/~jye02/Software/SLEP/), 
+ [GPML](http://www.gaussianprocess.org/gpml/code/matlab/doc/) and more. 
+ Future plans include to offer bindings to popular packets such as tensorflow and stan.
+
+### Free
+
+ * Shogun is **community based** and **non-commercial**.
+ * Licence: Shogun is currently released under GPLv3. 
+ We are working towards a BSD compatible repository and all newly added code is required to be BSD3 compliant.
+ This means companies can use the code without having to turn their code-base to GPL.
+
+### Good (is what we aim for)
+
+ * Shogun provides **efficient** implementation of all standard ML algorithms.
+  * Our implementation is [**competitive**](https://github.com/zoq/benchmarks) and we contribute to the benchmarking project by MLPack and Markus Edel (a framework to compare algorithms from different ML libraries).
+  * We also have exclusive **state-of-the art algorithms** (among others: efficient SVM implementations, Multiple Kernel Learning, kernel hypothesis testing and Krylov methods). 
+ All of these are supported by a collection of general purpose methods for evaluation, parameter tuning, preprocessing, serialisation and I/O.
+ * Shogun has an **extensive testing infrastructure**, making it reliable on dozens of OS setups.
+
+## <a name="useshoguntolearn"></a> Using Shogun to Learn
+
+Shogun is a community learning platform in every sense: **anyone can use the toolbox to learn about ML** and apply it to solve problems.
+
+ * Many **scientists** use Shogun as a tool for their research and it is [cited many hundred times on Google scholar](http://scholar.google.com/scholar?hl=en&q=shogun+toolbox&btnG=&as_sdt=1%2C33&as_sdtp=).
+ * We offer the Shogun cloud for **educational** purposes at universities and workshops (e.g. University College London *Introduction to Machine Learning*).
+
+Even more important, **everyone can gain skills by helping us to build Shogun**. We highly value our contributors and want to foster and develop them.
+
+ * We have mentored 33 **Google Summer of Code student projects** since 2011 (and counting).
+ Especially female students and students from diverse backgrounds are encouraged to apply.
+ * Many of our alumni students took their acquired skills to industry and we are proud to be part of their education.
+
+## <a name="shogunhistory"></a> Shogun History
+
+Shogun got initiated in 1999 by [Soeren Sonnenburg](http://sonnenburgs.de/soeren) and [Gunnar Raetsch](http://www.raetschlab.org/) (that's where the name *ShoGun* originates from). 
+The original focus was on large-scale kernel methods and bioinformatics. 
+Since then, Shogun has continuously been used for scientific research (a list of papers citing Shogun can be found [here](http://scholar.google.com/scholar?hl=en&q=shogun+toolbox&btnG=&as_sdt=1%2C33&as_sdtp=)) and massively expanded.
+
+Shogun's features now span the whole space of Machine Learning methods:
```

Comment:
"as I said before - please tell me more about the history so this can actually be a history. It doesn't need to be integrated into the features because everything that needs to be in features is already there"

This paragraph can just be removed.